internal/integration_test/fuzzcases/fuzzcases_test.go

```diff
@@ -437,3 +437,43 @@ func Test1777(t *testing.T) {
 		require.Equal(t, []uint64{18446626425965379583, 4607736361554183979}, res)
 	})
 }
+
+// Test2000 tests that v128.const i32x4 is not skipped when state is unreachable.
```

Comment:
I was not sure how the tests were originally numbered because from what I can see the fuzzer names the tests as a hex string... so I just took a high integer after 1777 (2000),

Reply:
can do. Since it's 3 tests I could change it to a string and use "1792a", "1792b", "1793c"

Reply:
sg